modules/data.atmosphere/R/met2CF.csv.R

```diff
@@ -35,18 +35,20 @@
 ##' in.prefix <- 'FLX_US-WCr_FLUXNET2015_SUBSET_HH_1999-2014_1-1'
 ##' outfolder <- '~/'
 ##' input.id <- 5000000005
-##' format <- query.format.vars(input.id=input.id,con)
+##' format <- query.format.vars(input.id=input.id,bety = bety)
 ##' start_date <- ymd_hm('200401010000')
 ##' end_date <- ymd_hm('200412312330')
 ##' PEcAn.data.atmosphere::met2CF.csv(in.path,in.prefix,outfolder,start_date,end_date,format,overwrite=TRUE)
 ##' }
 ##' @importFrom ncdf4 ncvar_get ncdim_def ncvar_add ncvar_put
+##' @importFrom lubridate year ymd_hm ymd_hms
+##' @importFrom udunits2 ud.is.parseable ud.convert ud.are.convertible
 met2CF.csv <- function(in.path, in.prefix, outfolder, start_date, end_date, format, lat = NULL, lon = NULL, 
                        nc_verbose = FALSE, overwrite = FALSE, ...) {
   library(PEcAn.utils)
   
-  start_year <- lubridate::year(start_date)
-  end_year   <- lubridate::year(end_date)
+  start_year <- year(start_date)
+  end_year   <- year(end_date)
```

Comment:
Yes, `foo <- package::foo()` is extremely bad practice. I think our style guide should express a clear, strong preference for `package::fun()` but accommodate `@importsFrom` for commonly used functions (rule-of-thumb -- 5 times in a single function?). If `@importsFrom` is used, we should consider whether or not to declare it once for the entire package via something like:

```r
#' @importsFrom PEcAn.utils, logger.severe, logger.warn
NULL
```

...as recommended by Hadley ([here][1] and [here][2]), or to require that it be declared in every function.

 We should _never_ use blanket `@imports` statements or `require/library` calls.

[1]: http://r-pkgs.had.co.nz/namespace.html#imports 
[2]: http://r-pkgs.had.co.nz/man.html#man-packages